docs/community/local-groups.mdx

```diff
@@ -105,6 +105,7 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Laguna Mesh](https://lagunamesh.com)
 - [Mission Viejo Mesh](https://missionviejomesh.org/)
 - [Meshtastic Bay Area Group](https://discord.gg/u2gxYu7RYQ)
+- [MontereyBayMesh](https://mrymesh.net/)
```

Comment:
I can't connect to this URL with HTTPS. Please change the link to HTTP or fix HTTPS on your web server.

Reply:
How embarrassing. Should be good to go now.